data/ecosystems/o/osmosis.toml

```diff
@@ -218,3 +252,30 @@ url = "https://github.com/osmosis-labs/wasmd"
 
 [[repo]]
 url = "https://github.com/osmosis-labs/wormhole"
+
+[[repo]]
+url = "https://github.com/forbole/big-dipper-2.0-cosmos"
+
+[[repo]]
+url = "https://github.com/BasVanDeGroep/osmosis-nft-demo"
+
+[[repo]]
+url = "https://github.com/pyramation/cosmology"
+
+[[repo]]
+url = "https://github.com/cosmos/chain-registry/"
+
+[[repo]]
+url = "https://github.com/CosmWasm/mesh-security"
+
+[[repo]]
+url = "https://github.com/Medniyy/Posthuman-Validator-Contributions-for-Osmosis-Chain"
+
+[[repo]]
+url = "https://github.com/blockpane/tenderduty"
```

Comment:
This is already covered in Cosmos ecosystem.

Reply:
This was built via a grant by the Osmosis grants program:
https://blockpane.medium.com/osmosis-grants-program-tenderduty-d4a167428aaf